File: src/main/java/com/howell/matt/test/SqsExamples.java
package com.howell.matt.test;

import com.amazonaws.regions.Regions;
import com.amazonaws.services.sqs.AmazonSQS;
import com.amazonaws.services.sqs.AmazonSQSClientBuilder;

import lombok.extern.slf4j.Slf4j;

/**
 * SQS examples.
 */
@Slf4j
public class SqsExamples {

    /**
     * Main.
     *
     * @param args args
     */
    public static void main(String[] args) {
        new SqsExamples().showQueues();
    }

    /**
     * Show SQS queues.
     */
    public void showQueues() {
        final var sqsClient = AmazonSQSClientBuilder.standard().withRegion(Regions.US_WEST_2).build();
        sqsClient.listQueues().getQueueUrls().forEach(log::info);
    }
}

File: src/main/kotlin/com/howell/matt/kotlin/test/SqsExamples.kt
package com.howell.matt.kotlin.test

import com.amazonaws.regions.Regions
import com.amazonaws.services.sqs.AmazonSQSClientBuilder

/**
 * SQS examples.
 */
class SqsExamples {

    val log = loggerFor(::SqsExamples.javaClass)

    /**
     * Show SQS queues.
     */
    fun showQueues() {
        val sqsClient = AmazonSQSClientBuilder.standard().withRegion(Regions.US_WEST_2).build()
        sqsClient.listQueues().queueUrls.forEach({ log.info(it) })
    }

    companion object {

        /**
         * Main.

         * @param args args
         */
        @JvmStatic fun main(args: Array<String>) {
            SqsExamples().showQueues()
        }
    }
}
